homeassistant/components/amcrest/sensor.py

```diff
@@ -89,32 +70,34 @@ def update(self):
         """Get the latest data and updates the state."""
         if not self.available:
             return
-        _LOGGER.debug("Updating %s sensor", self._name)
+        _LOGGER.debug("Updating %s sensor", self.entity_description.name)
```

Comment:
```suggestion
        _LOGGER.debug("Updating %s sensor", self.entity_description.key)
```

Reply:
Sorry this is not correct. I use name because that is set in __init__ and is needed to identify the sensor.

Reply:
It is, as in my suggestions, I've moved the existing key to name and added the dictionary key as the key to the description. This change represented the swap.

Reply:
Changed to .key, but the original was _name which was set to something else (now: f"{name} {self.entity_description.name}").